src/gui/widgets/text_box_base.hpp

```diff
@@ -331,6 +338,11 @@ class text_box_base : public styled_widget
 		text_.set_ellipse_mode(ellipse_mode);
 	}
 
+	void set_use_markup(bool use_markup) override
```

Comment:
Since these only shadow the base class I think you can just say `using styled_widget::set_use_markup` rather than declaring a member function, but I'm not sure.

Reply:
Actually - you can just omit the function altogether. You only need `using` if the derived class defines its own version with different qualifiers.

Reply:
removed function.